types/repo/AnonymousMessageRepo/AnonymousChannelsData.ts

```diff
@@ -20,6 +20,7 @@ export interface User {
   online: boolean;
   image: string;
   name: string;
+  username?: string;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `username` field has been added to the `User` interface. Ensure that all places where this interface is used are updated to handle this new optional field. If not handled properly, it could lead to undefined errors.

```diff
+  username?: string;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->